tools/affected-apps

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# A simple script to output the list of affected apps
+
+if AFFECTED_APPS=$(npx nx affected:apps --base=${AFFECTED_BASE} --plain | tail -1); then
```

Comment:
what does plain mean?

Reply:
Normally, `nx affected:apps` prints out beautified output, which is hard to parse. `--plain` switches the output format to simple strings separated by spaces.